frontend/app/src/components/dashboard/DashboardAssetTable.vue

```diff
@@ -23,22 +40,14 @@
       :custom-sort="sortItems"
       :custom-filter="assetFilter"
     >
-      <template #header.usdValue>
+      <template #header.usdValue="{ header }">
         <div class="text-no-wrap">
-          {{
-            $t('dashboard_asset_table.headers.value', {
-              symbol: currencySymbol
-            })
-          }}
+          {{ header.text }}
         </div>
       </template>
```

Comment:
I think that this and the same one that follows under it will have the same behavior as the default slot. I started removing them in other places but I forgot completely about this one. 

Maybe we should also proceed and remove them.